IAMOTZ/ievents
# #152917463 Create template for add-event page

#### What does this PR do?
Add the template for adding-event page
#### Description of Task to be completed?
Create template page for adding a new event
#### How should this be manually tested?
- clone the repo
- cd into the template directory
- open the addEvent.html with any browser of your choice
- the template should open in that browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152917463
#### Screenshots (if appropriate)
![just-now7](https://user-images.githubusercontent.com/26048536/32932863-4c5c5bce-cb69-11e7-8a30-903a4c8a88a2.png)
#### Questions:
None
